### PR TITLE
Don't use <div /> as a button

### PR DESCRIPTION
**Pull request by Eik-S**
As suggested by @rebloor this alignes the code to my changes in [the tutorial to this example](https://github.com/mdn/content/pull/21410)

### CODE REVIEW

**Comment by rebloor** [comment]
@Eik-S Please note https://github.com/mdn/content/pull/21410#issuecomment-1278495245

File: beastify/popup/choose_beast.html [hunk 1]
```diff
@@ -8,10 +8,10 @@
...
 <body>
   <div id="popup-content">
-    <div class="button beast">Frog</div>
-    <div class="button beast">Turtle</div>
-    <div class="button beast">Snake</div>
-    <div class="button reset">Reset</div>
+    <button class="button beast">Frog</button>
+    <button class="button beast">Turtle</button>
+    <button class="button beast">Snake</button>
+    <button class="button reset">Reset</button>
```
Comment (estelle) [inline beastify/popup/choose_beast.html:14]:
```suggestion
    <button>Frog</button>
    <button>Turtle</button>
    <button>Snake</button>
    <button type="reset">Reset</button>
```

File: beastify/popup/choose_beast.css [hunk 2]
```diff
@@ -7,6 +7,8 @@ html, body {
 }
 
 .button {
```
Comment (estelle) [inline beastify/popup/choose_beast.css:9]:
```suggestion
button {
```

**Review by estelle — CHANGES_REQUESTED** [review]
The class names aren't needed if using semantic elements. The CSS can be updated to read

```css
button {
  background-color: #E5F2F2;
}
button:hover {
  background-color: #CFF2F2;
}
[type="reset"] {
  background-color: #FBFBC9;
}
[type="reset"]:hover {
  background-color: #EAEA9D;
}

File: beastify/popup/choose_beast.html (hunk 1 above)
Comment (Eik-S) [reply beastify/popup/choose_beast.html:14]: i like it! Thanks.

**Comment by Eik-S** [comment]
Thanks @estelle 

I also adjusted the css selectors and a bug in the click listener.

**Review by estelle — APPROVED** [review]
Thank you!

**Pull request merged**